### DISCUSSION TITLE
I cannot log in with Permanent Password

### DISCUSSION THREAD

**Discussion by Danodes** [rustdesk/rustdesk#9005] [Q&A] [answered]
I'm installing the Rustdesk client on all my home computers, and I cannot log in with the permanent password in one of them.
I can change, enable or disable it, but it's the same.
The only way that I can log in is with the temporary password, and of course, only one time.
I tested with the other computers, the problem is the same only with that one.

I uninstalled and reinstalled it, turned it OFF as a service and on again, and deleted some folders in Windows/services, but nothing happened.

I'm using Anydesk only to log in and get the temporary password or simply, accept the connection, and then I connected with Rustdesk.

Do you have any ideas??

Thanks!

Daniel

**Comment by rustdesk** [comment, marked as answer]
There might be some file write problem in
`C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\config\RustDesk.toml`
Password is stored there. You can monitor this file, see what happens when password changes.

**Reply by rustdesk** [reply]
https://www.reddit.com/r/rustdesk/comments/1eoev69/rustdesk_resets_permanent_password/

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/9232

**Reply by Danodes** [reply]
Hi, the permanent password is working. I saw the changes in the file RustDesk.toml

And after some days, I tested again to log in with the permanent password and it worked without any problem. I don't understand why.
